packages/synapse-react-client/src/components/EntityAclEditor/EntityAclEditor.test.tsx

```diff
@@ -507,58 +518,6 @@ describe('EntityAclEditor', () => {
     ).toBeInTheDocument()
   })
 
-  it('displays an error on mutate failure', async () => {
```

Comment:
moved this test to the end, without changes.

This test dynamically updates a handler to return an error, but for some reason `server.restoreHandlers` didn't fix it. Not sure why, but moving the test fixes the issue